terra/test/src/helpers/vaa.ts

```diff
@@ -0,0 +1,117 @@
+import { soliditySha3 } from "web3-utils";
+
+const abi = require("web3-eth-abi");
+const elliptic = require("elliptic");
+
+export const TEST_SIGNER_PKS = [
+  "cfb12303a19cde580bb4dd771639b0d26bc68353645571a8cff516ab2ee113a0",
+];
+
+export function makeGovernanceVaaPayload(
+  governanceChain: number,
+  foreignChain: number,
+  foreignTokenBridge: string
+) {
+  return (
+    "000000000000000000000000000000000000000000546f6b656e427269646765" +
+    abi.encodeParameter("uint16", governanceChain).substring(2 + 62) +
+    "0000" +
+    abi.encodeParameter("uint16", foreignChain).substring(2 + (64 - 4)) +
+    foreignTokenBridge
+  );
+}
+
+export function makeTransferVaaPayload(
+  payloadType: number,
+  amount: string,
+  hexlifiedTokenAddress: string,
+  encodedTo: string,
+  toChain: number,
+  relayerFee: string,
+  additionalPayload: string | undefined
+): string {
+  const data =
+    abi.encodeParameter("uint8", payloadType).substring(2 + 62) +
+    // amount
+    abi.encodeParameter("uint256", amount).substring(2) +
+    // tokenaddress
+    hexlifiedTokenAddress +
+    // tokenchain
+    "0003" + // we only care about terra-specific tokens for these tests
+    // receiver
+    encodedTo +
+    // receiving chain
+    abi.encodeParameter("uint16", toChain).substring(2 + (64 - 4)) +
+    // fee
+    abi.encodeParameter("uint256", relayerFee).substring(2);
+
+  // additional payload
+  if (additionalPayload === undefined) {
+    additionalPayload = "";
+  }
+
+  // TODO: fix this
```

Comment:
what is there to fix here?

Reply:
I should have deleted this comment actually. There isn't anything to fix here anymore.